code/modules/mob/living/living.dm

```diff
@@ -526,6 +526,7 @@
 			for(var/S in mind.spell_list)
 				var/obj/effect/proc_holder/spell/spell = S
 				spell.updateButtonIcon()
+		src:clean_dna()
```

Comment:
No need to use src, just use clean_dna()
Also use .’s next time

Reply:
@Time-Green src: fixed some errors so that's why i used it

Reply:
Using : is a recipe for runtimes and maintainers will ree at you for it.